src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.tsx

```diff
@@ -79,18 +79,16 @@ const GeneOverview = () => {
         <span>{gene.stable_id}</span>
       </div>
 
-      {!!gene.name && (
-        <>
-          <div className={styles.sectionHead}>Gene name</div>
-          <div className={styles.geneName}>{gene.name}</div>
-        </>
-      )}
+      <div className={styles.sectionHead}>Gene name</div>
+      <div className={styles.geneName}>
+        {gene.name && gene.name.length ? gene.name : 'None'}
```

Comment:
Can we change this `gene.name && gene.name.length` to just `gene.name?.length`?

Reply:
I'm wondering whether we even need to check the length of gene.name. The backend should send either a string or a null as a value of this field. It shouldn't ever send an empty string.

Reply:
@imransl thanks for suggestion, it makes sense, i have made the changes.
@azangru whether the backend send us null or empty we will still need to make a check to display "None" unless we get the backend to send us the string "None" when it is empty. How long will it take them to do this? Also i feel like these kind of check is better done on our side just so that we dont have to ask the backend to make the change everytime. It will be easier and quicker for us to make the change :)

Reply:
> whether the backend send us null or empty we will still need to make a check to display "None" unless we get the backend to send us the string "None" when it is empty.

I don't understand...

We expect the backend to send us either a fully formed string if a gene has a name, or `null` if it doesn't. We do not expect them to ever send us an empty string.

Even having said that, an expression `{gene.name || 'None'}` should be just as effective as `{gene.name && gene.name.length ? gene.name : 'None'}`.